modules/Clinical/tests/test_conditional_requirements.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from pathlib import Path
+import sys
+import os
+import yaml
+from copy import deepcopy
+from linkml_runtime.loaders import yaml_loader
+from linkml_runtime.dumpers import yaml_dumper
+from htan_clinical.datamodel.clinical import ClinicalData
```

Comment:
Use the isort package to auto sort these package dependencies. (pep8 standards for package import ordering)